packages/blocks/src/api/registration.js

```diff
@@ -180,6 +180,17 @@ export function unstable__bootstrapServerSideBlockDefinitions( definitions ) {
 				serverSideBlockDefinitions[ blockName ].selectors =
 					definitions[ blockName ].selectors;
 			}
+
+			if (
+				serverSideBlockDefinitions[ blockName ]
+					.__experimentalAutoInsert === undefined &&
+				definitions[ blockName ].__experimentalAutoInsert
+			) {
+				serverSideBlockDefinitions[
+					blockName
+				].__experimentalAutoInsert =
+					definitions[ blockName ].__experimentalAutoInsert;
+			}
```

Comment:
@ockham I noticed this addition while resolving conflicts with #53807. This is basically a polyfill that won't be needed any more when WordPress Core itself starts exposing the `autoInsert` field. It would be nice to have a comment that says something like "this can be removed once the minimum supported version is 6.4". Other polyfills in this section also have such comments.

Reply:
Thanks for raising this, @jsnajdr! I'll add a comment to that effect.

Reply:
Ah, I see you already took care of this: https://github.com/WordPress/gutenberg/pull/53807/files#r1310211738

Thank you very much!